src/components/Header/Header.jsx

```diff
@@ -2,15 +2,17 @@ import React from "react";
 import { NavLink, useLocation } from "react-router-dom";
 import { useAuth } from "../../Context/auth-context";
 import "./Header.css";
-import {signOutHandler} from "../../services"
+import { signOutHandler } from "../../services";
+import { useWishlist } from "../../Context/wishlist-context";
 
 export default function Header() {
-  const { user,setUser  } = useAuth();
+  const { user, setUser } = useAuth();
   const { pathname } = useLocation();
+  const { wishlist } = useWishlist();
   const NavLinkStyles = ({ isActive }) => {
     return {
       fontWeight: isActive ? "bold" : "normal",
-    }; 
+    };
   };
```

Comment:
Maybe you could make a class for the active nav item.

Reply:
Nice Idea to work on✔